ZelBack/src/routes.js

```diff
@@ -1,912 +1,1770 @@
   app.get('/zelapps/temporarymessages', cache('30 seconds'), (req, res) => {
-    zelappsService.getZelAppsTemporaryMessages(req, res);
+    appsService.getAppsTemporaryMessages(req, res);
   });
   app.get('/zelapps/permanentmessages', cache('30 seconds'), (req, res) => {
-    zelappsService.getZelAppsPermanentMessages(req, res);
+    appsService.getAppsPermanentMessages(req, res);
   });
   app.get('/zelapps/globalappsspecifications', cache('30 seconds'), (req, res) => {
-    zelappsService.getGlobalZelAppsSpecifications(req, res);
+    appsService.getGlobalAppsSpecifications(req, res);
   });
   app.get('/zelapps/appspecifications/:appname?', cache('30 seconds'), (req, res) => {
-    zelappsService.getApplicationSpecificationAPI(req, res);
+    appsService.getApplicationSpecificationAPI(req, res);
   });
   app.get('/zelapps/appowner/:appname?', cache('30 seconds'), (req, res) => {
-    zelappsService.getApplicationOwnerAPI(req, res);
+    appsService.getApplicationOwnerAPI(req, res);
   });
   app.get('/zelapps/hashes', cache('30 seconds'), (req, res) => {
-    zelappsService.getZelAppHashes(req, res);
+    appsService.getAppHashes(req, res);
   });
   app.get('/zelapps/location/:appname?', cache('30 seconds'), (req, res) => {
-    zelappsService.getZelAppsLocation(req, res);
+    appsService.getAppsLocation(req, res);
   });
   app.get('/zelapps/locations', cache('30 seconds'), (req, res) => {
-    zelappsService.getZelAppsLocations(req, res);
+    appsService.getAppsLocations(req, res);
   });
   app.post('/zelapps/calculateprice', (req, res) => { // returns price in zel for both new registration of zelapp and update of zelapp
-    zelappsService.getAppPrice(req, res);
+    appsService.getAppPrice(req, res);
   });
   app.get('/zelapps/whitelistedrepositories', cache('30 seconds'), (req, res) => {
-    zelappsService.whitelistedRepositories(req, res);
+    appsService.whitelistedRepositories(req, res);
   });
   app.get('/zelapps/whitelistedzelids', cache('30 seconds'), (req, res) => {
-    zelappsService.whitelistedZelIDs(req, res);
-  });
-
-  // app.get('/explorer/allutxos', (req, res) => {
-  //   explorerService.getAllUtxos(req, res);
-  // });
-  // app.get('/explorer/alladdresseswithtransactions', (req, res) => {
-  //   explorerService.getAllAddressesWithTransactions(req, res);
-  // });
-  // app.get('/explorer/alladdresses', (req, res) => {
-  //   explorerService.getAllAddresses(req, res);
-  // });
-  // app.get('/explorer/zelnodetransactions', (req, res) => {
-  //   explorerService.getAllZelNodeTransactions(req, res);
-  // });
-  // filter can be IP, address, collateralHash.
-  app.get('/explorer/zelnodetxs/:filter?', cache('30 seconds'), (req, res) => {
-    explorerService.getFilteredZelNodeTxs(req, res);
-  });
-  app.get('/explorer/utxo/:address?', cache('30 seconds'), (req, res) => {
-    explorerService.getAddressUtxos(req, res);
-  });
-  app.get('/explorer/transactions/:address?', cache('30 seconds'), (req, res) => {
```

Comment:
Wrongly deleted?

Reply:
its still there but more down. 
APIs are now 'duplicatte' as of different path structure